[bug] AssertionError: 'AvailableIpView' should either include aserializer_classattribute, or override the get_serializer_class() method

While loading the docs endpoint:
`/api/v1/docs/`
Receiving the above error.

Full trace:
```
[04/Jan/2022 15:48:01] "GET /api/v1/docs/ HTTP/1.1" 200 2455 view's AvailableIpView raised exception during schema generation; usegetattr(self, 'swagger_fake_view', False)to detect and short-circuit this Traceback (most recent call last): File "\venv\lib\site-packages\drf_yasg\inspectors\base. py", line 42, in call_view_method return view_method() File "\venv\lib\site-packages\rest_framework\generics.p y", line 108, in get_serializer serializer_class = self.get_serializer_class() File "\venv\lib\site-packages\rest_framework\generics.py", line 122, in get_serializer_class assert self.serializer_class is not None, ( AssertionError: 'AvailableIpView' should either include a serializer_classattribute, or override theget_serializer_class()
method.
```

--- a/openwisp_ipam/api/views.py
+++ b/openwisp_ipam/api/views.py
@@ -157,6 +157,7 @@ class HostsSet:
 class AvailableIpView(ProtectedAPIMixin, IpAddressOrgMixin, RetrieveAPIView):
     subnet_model = Subnet
     queryset = IpAddress.objects.none()
+    serializer_class = serializers.Serializer
 
     def get(self, request, *args, **kwargs):
         subnet = get_object_or_404(self.subnet_model, pk=self.kwargs['subnet_id'])

